joomla/joomla-cms
# [4.0] We don't really need javascript for tooltips

Pull Request for Issue # .

### Summary of Changes
This was discussed in JAB and there was an agreement that JS is not needed for tooltips. Reasons:
- css can do that (cheeper, less code, less executable code)
- it's still accessible

Drawbacks:
- No more html elements in the tooltips


### Testing Instructions
Apply the patch and check the global configuration page (field: Use URL Rewriting)


### Expected result
Tooltips still rendering as before, but without jQuery and bootstrap dependency or any other js code 


### Actual result

![screen shot 2018-05-21 at 16 17 53](https://user-images.githubusercontent.com/3889375/40313186-4b23d550-5d15-11e8-95f5-012b03e8de2d.png)


### Documentation Changes Required
This PR also removes the archaic Mootools way of creating tooltips (class `hasTip`) since moo tools is not with us anymore 😓

@wilsonge 
@mbabker 
@laoneo 

@ciar4n you can take it from here...
